### ISSUE TITLE
SC1097 triggered by 'IFS== read'

### ISSUE THREAD

**Issue by deryni** [koalaman/shellcheck#1102]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1097
- My shellcheck version (`shellcheck --version` or "online"): 0.4.7 / online
- [X] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
IFS== read -r <<<foo
```

#### Here's what shellcheck currently says:

```none
$ shellcheck myscript
 
Line 3:
IFS== read -r <<<foo
   ^-- SC1097: Unexpected ==. For assignment, use =. For comparison, use [/[[.
```

#### Here's what I wanted or expected to see:

Nothing, this is a valid variable assignment of an equals sign to `IFS`.

That said, `IFS="="` does "solve" the problem but shouldn't be necessary here.

**Comment by Nightfirecat** [comment]
Duplicate of #1087, I believe. From that issue, the recommendation is to use the more explicit `="="` for clarity or to use an ignore statement.
